.gitignore

```diff
@@ -13,6 +13,7 @@ config/database.yml
 # Ignore sqlite3 databases
 **/*.sqlite*
 
-# Ignore and env files
+# Ignore general local files
+Guardfile
```

Comment:
I think this file belongs to this pull request, but the previous one you opened.